### PR TITLE
[SDPA-1815] Process webform event using decode entities

### PR DESCRIPTION
**Pull request by enzolutions**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-1815

Related PRs: 
* Content PR:  https://github.com/dpc-sdp/content-vic-gov-au/pull/482
### Changed

1. Process webform event using decode entities

### CODE REVIEW

File: tide_event.module [hunk 1]
```diff
@@ -123,38 +123,38 @@ function tide_event_webform_submission_presave($submission) {
     $event_details_data = [
       'type' => 'event_details',
       'field_paragraph_link' => [
-        'uri' => Html::escape($submission_data['website_url_for_booking']),
+        'uri' => Html::decodeEntities(Html::escape($submission_data['website_url_for_booking'])),
```
Comment (sonnykt) [inline tide_event.module:126]: @enzolutions Why do we need to escape/decode input data? Those utility functions from Html are for sanitising output.

**Review by sonnykt — CHANGES_REQUESTED** [review]

File: tide_event.module (hunk 1 above)
Comment (enzolutions) [reply tide_event.module:126]: @sonnykt @GROwen requested to Sanitize the input of the user from FE, I am open to any suggestions

Comment (sonnykt) [reply tide_event.module:126]: @enzolutions @GROwen pls refer to https://www.drupal.org/docs/develop/security/why-does-drupal-filter-on-output

Comment (enzolutions) [reply tide_event.module:126]: I agree with you and that document, I already discuss that with @GROwen, so Guy is your call. we could remove those? and trust in Drupal Entity Saving validation for user Input?

**Comment by anthony-malkoun** [comment]
This is the wrong approach to me. As discussed in the JIRA ticket we should not be sanitising input and will need to be careful on render (either from the frontend, but probably more correctly on JSONAPI output) that output is sanitised.

**Pull request closed without merging**